app/views/members/_stats.html.haml

```diff
@@ -2,34 +2,34 @@
 
 %ul.list-inline
   %li
-    - unless member.plantings.empty?
+    - if !member.plantings.empty?
```

Comment:
Cool, thanks - I was confused about why you were changing some `if !`s to `unless`es and some `unless`es to `if !`s.